app/components/web-components/au-dropdown/item.hbs

```diff
@@ -3,6 +3,10 @@
     <a class="auk-menu__item active {{this.skin}}" href={{@href}} ...attributes>
       {{yield}}
     </a>
+  {{else if @textOnly}}
+    <div class="auk-menu__item--text-only" ...attributes>
+      {{yield}}
+    </div>
```

Comment:
Ik zou de text-only versie eerder de default fallback laten zijn.
Dus een logica in de template:
- if `@href`, render `<a>` 
- else if `@route`, render `<LinkTo>`
- else if `@onClick`, render clickable element
- else, render text only

Reply:
Probleem is als we dit deden dat er dan veel dingen braken. Dus misschien beter voor een nieuw ticket?

Reply:
![image](https://user-images.githubusercontent.com/16088932/117812753-7dad9880-b262-11eb-806e-580c23a4d1bf.png)
Wel klikbaar maar niet meer met juiste css
```
{{#if @agendaitem.retracted}}
              <WebComponents::AuDropdown::Item {{on "click" (fn this.advanceAgendaitem @agendaitem)}} data-test-agendaitem-controls-action-advance>
                {{t "not-postpone"}}
              </WebComponents::AuDropdown::Item>
            {{else}}
```

Deze zou dan bijv zo'n @onClickAction moeten krijgen voor die on click dan zou die de juiste css krijgen

Reply:
@erikap default text-only is geen goede optie
In de meeste gevallen wordt die AuDropdown::Item nog gebruikt met een {{on "click" fn...
vb:
```
<WebComponents::AuDropdown::Item {{on "click" this.toggleIsVerifying}}
```
In dat geval kwamen we in de huidige default: een link-to zonder een route, waar een actie aan is verbonden.

Reply:
Ik ben niet helemaal fan van die nieuwe 'onClickAction', {{ on "click" }} lijkt me beter, minder tussenstappen.

Zelfs als we dat wel zouden gebruiken, het gebruik van <div> willen we niet, dat klopt niet met de styleguide
http://development.kaleidos-prototype.mono.digital/styleguide/auk-dropdown.html#

Reply:
en die div zorgt er ook voor dat je een foute mouse pointer hebt.
met een a of linkto heb je een wijzende vinger pointer
met div heb je die pointer niet

Reply:
Om deze story niet te laten ontsporen inderdaad dan best voor de `@textOnly` option gaan.

`div` is inderdaad niet juist. Het moet dan een `button` zijn. De HTML die gegenereerd wordt nu voor die gevallen met een on-click handler zijn niet valid. Anchor tags moeten gebruikt worden voor hyperlinks terwijl er voor deze cases geen navigatie gebeurd maar een actie uitgevoerd wordt. Qua accessibility is dit niet ok.

Ik heb ook een voorkeur voor `{{ on "click" }}` omdat we dan dichter bij native Ember/HTML blijven. Maar als er geen manier is (zelf nog niet naar gezocht) om te weten of er een on-click handler gezet is op de component, zie ik nog niet hoe we de cases correct kunnen afhandelen in de component.

Reply:
Die pointer kan wel css gewijs toegevoegd worden aan die div , maar ik weet niet of dat gewenst is